modules/world.py

```diff
@@ -63,6 +66,21 @@ def exit(self):
             return True
         else:
             return False
+			
+    def pick_up_item(self):
+        key_x = self.data["key"]['x']
+        key_y = self.data["key"]['y']
+        gold_x = self.data["gold"]['x']
+        gold_y = self.data["gold"]['y']
+        x, y = self.locate("player")		
+               
+        
```

Comment:
There is locate(), but what it really looks like you are trying to figure out is what is in the same tile as the player.  Luckily that exists too :)

Room.items(x,y) will tell you everything at those co-ordinates including the player